File: src/test/java/io/github/binout/soccer/domain/date/LeagueMatchDateTest.java
package io.github.binout.soccer.domain.date;

import org.junit.Test;

import java.time.LocalDate;

import static org.assertj.core.api.Assertions.assertThat;

public class LeagueMatchDateTest {

    @Test
    public void date_has_no_present_by_default() {
        assertThat(new LeagueMatchDate(LocalDate.now()).nbPresents()).isZero();
    }

}

File: src/test/kotlin/io/github/binout/soccer/domain/date/LeagueMatchDateTest.kt
package io.github.binout.soccer.domain.date

import org.junit.jupiter.api.Test

import java.time.LocalDate

import org.assertj.core.api.Assertions.assertThat

class LeagueMatchDateTest {

    @Test
    fun date_has_no_present_by_default() {
        assertThat(LeagueMatchDate(LocalDate.now()).nbPresents()).isZero()
    }

}